demo/chbench/mzcompose.yml

```diff
@@ -42,6 +42,22 @@ services:
       # 1000 was chosen by fair dice roll
       - DIFFERENTIAL_EAGER_MERGE=1000
       - DEFAULT_PROGRESS_MODE=${DEFAULT_PROGRESS_MODE}
+  materialized-release:
+    image: materialize/materialized:${MATERIALIZED_RELEASE_TAG:-v0.5.3}
+    ports:
+     - *materialized
+    init: true
+    command: -w ${MZ_THREADS:-1}
+    environment:
+      # you can for example add `pgwire=trace` or change `info` to `debug` to get more verbose logs
+      - MZ_LOG=pgwire=debug,info
+      # We want this to eventually count up to the size of the largest batch in an
+      # arrangement. This number represents a tradeoff between proactive merging (which
+      # takes time) and low latency.
+      #
+      # 1000 was chosen by fair dice roll
+      - DIFFERENTIAL_EAGER_MERGE=1000
+      - DEFAULT_PROGRESS_MODE=${DEFAULT_PROGRESS_MODE}
```

Comment:
not sure if this is already on your list of things to bench, but playing with these knobs is probably one of the things we most need to understand.

Reply:
Yes, this is on the list of things. I will file a ticket to make sure it's tracked. Thanks!